.claude/hooks/verify-migrations.sh

```
@@ -0,0 +1,73 @@
+  *"git commit"* | *"git push"*) ;;
+  *) exit 0 ;;
+esac
+
+[ "${SKIP_MIGRATION_CHECK:-}" = "1" ] && exit 0
+cd "$CLAUDE_PROJECT_DIR" || exit 0
+
+# 변경된 마이그레이션 탐지 — staged(commit) + 미푸시 커밋(push)
+CHANGED=$(
+  {
+    git diff --cached --name-only
+    git diff --name-only @{u}..HEAD 2>/dev/null
```

Shellcheck (0.11.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)